edgelet/edgelet-kube/src/module/trust_bundle.rs

```diff
@@ -212,8 +212,8 @@ mod tests {
                     "items": [
                         {
                             "metadata": {
-                                "name": constants::PROXY_CONFIG_TRUST_BUNDLE_NAME,
-                                "namespace": "my-namespace",
+                                "name": "device1-iotedged-proxy-trust-bundle",
```

Comment:
should these be constants and share across tests?

Reply:
Good idea.